packages/frontend/scss/login/_login.scss

```
@@ -201,25 +201,29 @@ div.delta-form-group {
   }
 }
 
-progress.delta-progress-bar {
+div.delta-progress-bar {
```

```suggestion
.delta-progress-bar {
```

I think `progress` was specified purely for the reader, to say that it is a special element. I think it's better not to specify `div` explicitly if, say, we happen to go from `div` to `span`.

Though feel free to ignore this suggestion, it's just a minor nitpick.